tools/cdk-build-tools/bin/cdk-test.ts

```diff
@@ -9,12 +9,6 @@ async function main() {
   const args = yargs
     .env('CDK_TEST')
     .usage('Usage: cdk-test')
```

Comment:
Looks like this is from another PR

Reply:
I needed this to build the damn thing. If you ship the other PR then the diff will go away :)